homeassistant/components/intellidrive/entity.py

```diff
@@ -0,0 +1,56 @@
+"""Entity for the opengarage.io component."""
+from __future__ import annotations
+
+from homeassistant.core import callback
+from homeassistant.helpers.device_registry import CONNECTION_NETWORK_MAC
+from homeassistant.helpers.entity import DeviceInfo, EntityDescription
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, STATUSDICT_SERIALNO, STATUSDICT_VERSION
+from .coordinator import ReisingerCoordinator
+
+
+class IntelliDriveEntity(CoordinatorEntity[ReisingerCoordinator]):
+    """Representation of a IntelliDrive entity, which is used as a base entity for future entities."""
+
+    def __init__(
```

Comment:
has_entity_name is mandatory for new integrations, I think you can simply enable it: https://developers.home-assistant.io/docs/core/entity#has_entity_name-true-mandatory-for-new-integrations

```suggestion
class IntelliDriveEntity(CoordinatorEntity[ReisingerCoordinator]):
    """Representation of a IntelliDrive entity, which is used as a base entity for future entities."""

    _attr_has_entity_name = True

    def __init__(
```